### PR TITLE
boilerplate with Remix, Chakra-UI, Visx

### PR DESCRIPTION
**Pull request by SiriusLL**
☕️

### CODE REVIEW

File: app/root.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+} from "remix";
+import type { MetaFunction } from "remix";
+import { ChakraProvider } from "@chakra-ui/react";
+
+
+// export const meta: MetaFunction = () => {
+//   return { title: "New Remix App" };
+// };
+
+
```
Comment (mattesats) [inline app/root.tsx:17]: It's always a good practice to remove commented code or code that isn't being used.

File: remix.config.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * @type {import('@remix-run/dev').AppConfig}
+ */
+module.exports = {
+  ignoredRouteFiles: [".*"]
+  // appDirectory: "app",
+  // assetsBuildDirectory: "public/build",
+  // serverBuildPath: "build/index.js",
+  // publicPath: "/build/",
+  // devServerPort: 8002
```
Comment (mattesats) [inline remix.config.js:10]: Can you explain why this is commented? These kinda look like the default options...

**Review by mattesats — CHANGES_REQUESTED** [review]
Looks good! Just a couple questions. @SiriusLL

File: app/root.tsx (hunk 1 above)
Comment (SiriusLL) [reply app/root.tsx]: Check

File: remix.config.js (hunk 2 above)
Comment (SiriusLL) [reply remix.config.js:10]: These are commented by default, I could not find much information as to why.

Comment (aquanow-greg) [reply remix.config.js:10]: Ok. You can just remove these lines then.

**Review by mattesats — APPROVED** [review]

**Comment by mattesats** [comment]
Closes #3

**Pull request merged**